frontend/app/assets/javascripts/extent_calculator.js

```diff
@@ -1,13 +1,26 @@
 $(function () {
-  function ExtentCalculatorForm() {}
+  function ExtentCalculatorForm() { }
 
   ExtentCalculatorForm.prototype.init_form = function () {
     $('.create-extent-btn').on('click', function (event) {
+      if (
+        !(
+          $('#extent_portion_').val() &&
+          $('#extent_number_').val() &&
+          $('#extent_extent_type_').val()
+        )
+      ) {
+        alert('Please ensure that all required fields contain a value.');
```

Comment:
This alert creates important feedback to the user in hardcoded English instead of some potential other locale. Maybe make it a .ERB file and throw the translation function in there?

<img width="1624" alt="ANW-1771 alert" src="https://github.com/archivesspace/archivesspace/assets/3411019/6b4efe72-4f66-4ef9-8aa9-7274694ecb2c">

Reply:
Ah, good point. I think I'll drop the translation into a span or something in the ERB so I can access it from the Javascript.

Reply:
Yes much better than renaming the file!